fdbserver/storageserver.actor.cpp

```diff
@@ -3457,9 +3457,9 @@ ACTOR Future<Void> metricsCore( StorageServer* self, StorageServerInterface ssi
 					self->metrics.splitMetrics( req );
 				}
 			}
-			when (GetPhysicalMetricsRequest req = waitNext(ssi.getPhysicalMetrics.getFuture())) {
+			when (GetStorageMetricsRequest req = waitNext(ssi.getStorageMetrics.getFuture())) {
 				StorageBytes sb = self->storage.getStorageBytes();
-				self->metrics.getPhysicalMetrics( req, sb );
+				self->metrics.getStorageMetrics( req, sb, self->counters.bytesInput.getRate() );
```

Comment:
The time interval for counters are governed by when the last time it was traced. So if you call getRate() immediately after a StorageMetrics was logged, you will only get the bytes input for a very small amount of time. Maybe make a new smoother for tracking BytesInput.

Reply:
Yeah I had considered a smoother, but since I hadn't given much consideration yet to the feature that would actually use the value, I decided to defer making any decisions about it until feature implementation time. I'm happy to add it now if we know what we want it to do.

Reply:
Will defer